rosa/rosa-wrapper.py

```diff
@@ -106,6 +106,24 @@ def _install_addons(rosa_cmnd,cluster_id,addons):
             logging.error(addon_stderr.strip().decode("utf-8"))
         # TODO: control addon is installed with: rosa list addons -c <<cluster_name>>
 
+def _expiration(rosa_cmnd,cluster_id,expiration):
```

Comment:
(nitpick) It might be useful to rename this function to better represent its purpose without having to look at its implementation. Example name: `_extend_cluster_expiration()`

Reply:
done